shared/actions/tracker.js

```diff
@@ -425,7 +425,7 @@ function serverCallMap (dispatch: Dispatch, getState: Function, skipPopups: bool
 
   const requestIdle = f => {
```

Comment:
@gabriel 

We have this which is essentially `requestIdleCallbackForIdentify`.

would you want it renamed and a comment placed above the timeout?